soap/src/java/com/zimbra/soap/account/message/GetInfoResponse.java

```diff
@@ -504,6 +512,14 @@ public String getSoapURL() {
         return Prop.toMultimap(props, userPropKey);
     }
 
+    public Boolean getIsTrackingIMAP() {
```

Comment:
I would change this to a boolean as toBool specifying a default will never return null.